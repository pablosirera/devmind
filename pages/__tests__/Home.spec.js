import { shallowMount } from '@vue/test-utils'
import HomeView from '@/pages/index.vue'

describe('Home View', () => {
  const build = () => {
    const wrapper = shallowMount(HomeView)

    return {
      wrapper,
    }
  }

  test('should be render', () => {
    const { wrapper } = build()
    expect(wrapper.vm).toBeTruthy()
  })

  test('should match snapshot', () => {
    const { wrapper } = build()
    expect(wrapper.element).toMatchSnapshot()
  })
})
