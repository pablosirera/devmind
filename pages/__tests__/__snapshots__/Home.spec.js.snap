// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home View should match snapshot 1`] = `
<div>
  <colorswitch-stub />
   
  <logo-stub />
</div>
`;
